### DISCUSSION TITLE
Will not function on head unit until initiated from phone

### DISCUSSION THREAD

**Discussion by DRB523** [agronick/aa-torque#106] [Q&A]
I have successfully got AA Torque running on my phone and it can be launched on the head unit in my vehicle (wirelessly) but it will not retrieve any data until I go to the app on the phone and launch it and click the view gauges button on the top of the settings screen.  I have to go through this entire process every time I start/restart my vehicle.  Any thoughts?

Also, not quite as important, but whenever I view the settings screen on my phone while I am connected to my vehicle AA, the settings screen is white, not black like it normally is.  Any thoughts?

Thanks

**Comment by agronick** [comment]
Check your power management settings. Let Torque run in the background. Theme changes based off dark mode being on.

**Comment by DRB523** [comment]
Thank you for the quick response agronick.  I have been through my settings and do not see how to let Torque run in the background.  And for clarification, are you referring to the Torque app or your AA Torque?

Thank you

**Comment by agronick** [comment]
The Torque app. But that is strange since the view gauges button runs the same code that runs when it is displaying on the headunit. Sometimes it takes a few seconds to connect. You can speed up the process by telling Torque which protocol to use so it doesn't have to search every time. Do that in the vehicle settings in Torque.
